frontend/src/pages/project.jsx

```diff
@@ -0,0 +1,130 @@
+import * as React from 'react';
+import {
+	Button, Typography, FormControl, FormControlLabel, Paper, Checkbox, Input, InputLabel, SnackbarContent, Grid, TableBody, Table, TableRow, TableCell
+} from '@material-ui/core';
+import { Theme } from '@material-ui/core/styles/createMuiTheme';
+import { createStyles, WithStyles } from '@material-ui/core/styles';
+import withStyles from '@material-ui/core/styles/withStyles';
+import { TransformRounded as Logo, LockOutlined as LockOutlinedIcon, Error as Icon } from '@material-ui/icons';
+import { Response, Project as ProjectFetch } from "../utils/connect"
+import { RouteComponentProps } from 'react-router-dom';
+import { Project, GetProjectResponse, ProjectDetails } from "../utils/connect"
+import { ModelsSearchComponent } from '../components/ModelsSearch'
+
+const styles = (theme) =>
+	createStyles({
+		cover: {
+			width: "100%",
+			padding: "40px 0",
+			display: "flex",
+			flexDirection: "column",
+			justifyContent: "space-between"
+		},
+		coverItem: {
+			marginBottom: theme.spacing.unit
+		},
+		button: {
+			margin: theme.spacing.unit,
+		},
+		row: {
+			display: "flex",
+			flexDirection: "row",
+			marginTop: "10px",
+			"& *:first-child":{
+				marginLeft:0
+			}
+		}
+	});
+
+class ProjectComponent extends React.Component {
+	lastReq = "";
+	constructor(props) {
+		super(props);
+		this.state = {
+			status: "loading",
+			id: 0,
+			name: "please wait",
+			description: "please wait",
+			repoUrl: "please wait",
+			allParameters: ["please wait"],
+			allHiperParameters: ["please wait"],
+			allModelTags: ["please wait"],
+			allModelNames: ["please wait"],
```

Comment:
Not sure if we would want to store `allModelNames` in the `Project` component. 
What is the intended use case of this array?

Reply:
This was intended for a few names as i expected 'name'  to identify line of models instead of being description for particular model. I insist on this approach.